corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -486,6 +486,7 @@
         <![endif]-->
         
         {% include 'style/includes/keyboard_shortcuts.html' %}
+        {% block keyboard_shortcuts %}{% endblock keyboard_shortcuts %}
```

Comment:
Good catch, the block in keyboard_shortcuts should be removed (nothing extends that template). Added a commit. Fwiw, django didn't seem to complain about having the block defined twice.